go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/oc-mirror
 
-go 1.19
+go 1.20
```

Comment:
Is this go bump necessary?

Reply:
It is not necessary anymore since now there is a 1.19 image with the CVE fix. I will revert it back to 1.19